### ISSUE TITLE
Not Able to Connect

### ISSUE THREAD

**Issue by michaelmagan** [hsand/pia-wg#3]
I used the tool to generate the config and it said everything worked but when I put this in my wireguard set up it doesn't work...

```
[Interface]
Address = 10.38.134.8
PrivateKey = <>
DNS = 10.0.0.243,10.0.0.242

[Peer]
PublicKey = <>
Endpoint = 46.246.3.218:1337
AllowedIPs = 0.0.0.0/0
PersistentKeepalive = 25
```

**Comment by hsand** [comment]
Could you test with a few different PIA servers, and see if you're having the same issue? Just to rule out an issue on PIA's side.

**Comment by michaelmagan** [comment]
@hsand Yes. I've already done that and confirmed another server works. It's possible it's restricted to this server. Thanks for creating this utility. It's pretty great.

I'm going to spend sometime testing more servers to see what happens.

**Comment by dkrutsko** [comment]
Yeah I'm having the same issue where I'm able to connect to `CA-Toronto` but not get internet access. On the other hand, accessing `US-New-York` works fine. I'll see if I can try and figure out what the issue seems to be. Maybe some of those servers just don't support WG properly?

**Comment by fracai** [comment]
I'm encountering the same behavior (connection succeeds, but no Internet access).
The servers I tried last night are: 
- CA-Toronto
- Netherlands
- UK-London
- US-New-York

Any debugging tips?

**Comment by chimeforest** [comment]
I'm having the same problem.
Using the wireguard app on windows, and a GL.iNet router.
Windows can "Activate" all of them, but no internet access. ~~Router can only connect to the one successful one listen below.~~
There's probably not a whole lot we can do, but here are the servers I tried to connect to...

Successful:
- ~~US-Houston~~

Unsuccessful:
- Bahamas
- CA-Montreal
- CA-Toronto
- Greenland
- Mexico
- US-Baltimore
- US-Chicago
- US-East
- US-Houston
- US-New-York
- US-Washington-DC
- US-Wilmington
- Venezuela

Edit: Now Houston is not working either =/

**Comment by weller9999** [comment]
It seems that the script hardcodes the IP in the conf file.  I can connect but get no traffic.  I then noticed that the IP changes every 1 second:
This is New Zealand but every site is constantly changing IPs, so then the endpoint will have the wrong IP.

for i in {1..10}; do curl https://serverlist.piaservers.net/vpninfo/servers/v4 2>/dev/null|tr , "\n"|egrep "id.:|wg"|\grep nz -A1|tail -1; sleep 1; done
"wg":[{"ip":"179.61.240.179"
"wg":[{"ip":"179.61.240.108"
"wg":[{"ip":"179.61.240.179"
"wg":[{"ip":"179.61.240.142"
"wg":[{"ip":"179.61.240.159"
"wg":[{"ip":"179.61.240.159"
"wg":[{"ip":"179.61.240.164"
"wg":[{"ip":"179.61.240.134"
"wg":[{"ip":"179.61.240.160"
"wg":[{"ip":"179.61.240.170"

Thoughts?

**Comment by Ph0rk0z** [comment]
I am able to connect but DNS doesn't work. I can go to say 1.1.1.1 by IP but all pings to the pia dns servers fail and I haven't been able to use any other one.

**Comment by ZenoBell** [comment]
I selected a streaming server and i had the same issue. Changing to something else solved the issue so i guess some of them are not working as expected.
